src/cogent3/util/misc.py

```diff
@@ -130,10 +131,20 @@ def bytes_to_string(data):
     return data
 
 
+def zip_open(filename, mode="r", **kwargs):
+    """open a zip-compressed file"""
+    if mode == "rt":
```

Comment:
Why block mode "rt"?

Reply:
Hi Gavin, I added this conversion simple because Python Zip only supports "r" and "w" modes,
```
        if mode not in {"r", "w"}:
            raise ValueError('open() requires mode "r" or "w"')
```
but we have got mode "rt" as default mode in function `open_`

Reply:
OK, let's just trim off trailing "t".